swift/ql/lib/codeql/swift/controlflow/internal/ControlFlowElements.qll

```diff
@@ -0,0 +1,162 @@
+private import swift
+
+newtype TControlFlowElement =
```

Comment:
Cached?

Reply:
Yeah, this should definitely be cached. Thanks!